src/RevLoader.cc

```diff
@@ -10,6 +10,7 @@
 
 #include "RevLoader.h"
 #include "RevMem.h"
+#include <_types/_uint32_t.h>
```

Comment:
Why this strange header instead of `<cstdint>`?

Reply:
I like to be different

Reply:
Jk changed it